deployment/environment/devenv/chain.go

```diff
@@ -94,58 +96,63 @@ func (c *ChainConfig) SetDeployerKey(pvtKeyStr *string) error {
 
 func NewChains(logger logger.Logger, configs []ChainConfig) (map[uint64]deployment.Chain, error) {
 	chains := make(map[uint64]deployment.Chain)
+	g := new(errgroup.Group)
 	for _, chainCfg := range configs {
-		selector, err := chainselectors.SelectorFromChainId(chainCfg.ChainID)
-		if err != nil {
-			return nil, fmt.Errorf("failed to get selector from chain id %d: %w", chainCfg.ChainID, err)
-		}
-		// TODO : better client handling
-		var ec *ethclient.Client
-		for _, rpc := range chainCfg.WSRPCs {
-			ec, err = ethclient.Dial(rpc.External)
+		g.Go(func() error {
```

Comment:
I think you might get var override error without this
```suggestion
                chainCfg:=chainCfg
		g.Go(func() error {
```